### DISCUSSION TITLE
Cannot read excel file into Pandas data frame

### DISCUSSION THREAD

**Discussion by sdaniels7** [community/community#142214] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone!

When I try to import a file using Pandas library, I receive the following error:

Input:
import pandas as pd
df = pd.read_excel(r"C:\Users\Administrator\Desktop\Module6.xlsx")
df.shape

Output:
[Errno 2] No such file or directory: '[C:\\Users\\Administrator\\Desktop\\Module6.xlsx](file:///C://Users//Administrator//Desktop//Module6.xlsx)'

It seems to add an extra "\" to the path and then cannot find it. Has anyone encountered/resolved this issue?
![Errno2](https://github.com/user-attachments/assets/8c5902a2-42ae-47e9-aa39-0e1169d7f431)

**Comment by wlb-tycoon** [comment]
I think you're encountering those errors because the file path is incorrect or the file does not exist at the specified location.
Let me share a few ways you can take to resolve this issue:
1. **Check the File Path:** Ensure that the file `Module6.xlsx` actually exists at the path `C:\Users\Administrator\Desktop\`. You can do this by navigating to that location in Windows Explorer.
2. **Double Backslashes:** If you're not using a raw string, you can escape the backslashes by using double backslashes:
```
df = pd.read_excel("C:\\Users\\Administrator\\Desktop\\Module6.xlsx")
```
3. **Check Permissions:** Make sure that you have permission to access the file. Sometimes, permission issues can also lead to such errors.

4. **Check for Typographical Errors:** Make sure there are no typos in the file name or path.

5. **Try Using an Alternative Method:** If the above doesn't work, you can try using os.path to construct the path, which can sometimes help avoid issues:
```
import os
import pandas as pd

file_path = os.path.join("C:", "Users", "Administrator", "Desktop", "Module6.xlsx")
df = pd.read_excel(file_path)
```
6. Check for Hidden Extensions: In Windows, sometimes file extensions are hidden. Ensure that the file is indeed named `Module6.xlsx` and not something like `Module6.xlsx.xlsx`.
